db/re/item_db.conf

```diff
@@ -61939,6 +61940,11 @@ item_db: (
 	Slots: 1
 	Loc: 256
 	View: 525
+	Script: <"
+		bonus bDex, 2;
+		skill TF_DOUBLE, 5;
```

Comment:
No, it will have level 10 Double Attack, but will be able to do a Double Attack with 50% chance with every weapon he equips, such as Swords, Bow and Axe, and when being an Assassin, will be able to do Double Attack with katars, just because have equipped that kind of gear, otherwise they couldn't do the Double Attack (even if learns the skill through gear), because Double Attack only works with daggers.